### PR TITLE
fixes output and doc

### PR DESCRIPTION
**Pull request by sk-sahu**
## Change log   
- Output for `cloudos::extract_participants()` now in a R data-frame. 
- Documentation of each function improved and examples added.

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -5,3 +5,4 @@
 ^_pkgdown\.yml$
 ^docs$
 ^pkgdown$
+^local*
```
[the remark this answers is not in the document]
Comment (sk-sahu) [reply .Rbuildignore:8]: Some files for testing which I have locally having secrets (apikey, tokens)

File: R/list_cohorts.R [hunk 2]
```diff
@@ -2,30 +2,30 @@
...
 #'
 #' @return A list.
 #'
-# @examples 
-# list_cohorts(baseurl= "https://cloudos.lifebit.ai", 
-#              auth = "Bearer ***token***",
-#              teamid = "***teamid***")
-#'
+#' @examples
+#' \dontrun{
```
[the remark this answers is not in the document]
Comment (sk-sahu) [reply R/list_cohorts.R:15]: Yes from [roxygen2 vignettes](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html)

image

File: tests/testthat/test-extract.R [hunk 3]
```diff
@@ -0,0 +1,3 @@
+test_that("dummy test - length computed correctly", {
```
[the remark this answers is not in the document]
Comment (sk-sahu) [reply tests/testthat/test-extract.R:1]: For now we don't have a real test case. I'll be working on this in future iterations.

**Comment by sk-sahu** [comment]
@cgpu Comments done. Please have a final look. This time I'll squash and merger :smile:

File: tests/testthat/test-extract.R (hunk 3 above)
[the remark this answers is not in the document]
Comment (cgpu) [reply tests/testthat/test-extract.R:1]: sounds good, at least the placeholder test is replaced 💪

**Comment by cgpu** [comment]
@sk-sahu feel free to delete the branch since this is merged now to keep it clean and tidy.

**Pull request merged**